tests/unit/mathematics/linalg/operations/Eigen3_operations_unittest.cc

```diff
@@ -11,189 +11,269 @@ using namespace shogun;
 using namespace linalg;
 using namespace Eigen;
 
-TEST(LinalgBackendEigen, SGVector_add)
+// Tolerance values for tests
+template <typename T>
+// default tolerance
+T get_epsilon()
 {
-	const float64_t alpha = 0.3;
-	const float64_t beta = -1.5;
+	return 0;
+}
+template <>
```

Comment:
Any chance you can make this more compact using https://en.cppreference.com/w/cpp/types/numeric_limits/epsilon

Reply:
I think taking the machine epsilon and then dropping two orders of magnitude is a reasonable idea